src/common/scripts/vnc_startup.sh

```diff
@@ -106,11 +108,13 @@ if [[ $DEBUG == true ]] || [[ $1 =~ -t|--tail-log ]]; then
     tail -f $STARTUPDIR/*.log $HOME/.vnc/*$DISPLAY.log
 fi
 
+xhost 127.0.0.1
```

Comment:
slightly insecure (by design), containers at same IP address (i.e. as in a kubernetes pod) are trusted by default to display to this XServer)